### DISCUSSION TITLE
Game crashes upon starting new game or load save

### DISCUSSION THREAD

**Discussion by Thatoneunodude** [X2CommunityCore/xcom2-launcher#360] [Help]
**1.5.0:** 

**Description**
Game Crashes to desktop upon loading a save or starting the first mission even if no mods are loaded.
starting from the 2k launcher results in no issue.
other troubleshoot methods didnt work so far.

[AML.log](https://github.com/X2CommunityCore/xcom2-launcher/files/12707163/AML.log)
[settings.odt](https://github.com/X2CommunityCore/xcom2-launcher/files/12707171/settings.odt)

**Comment by Iridar** [comment]
You might need to launch AML as admin.
